docs/framework/install/guide-for-developers.md

```diff
@@ -11,188 +11,186 @@ ms.assetid: daf9d9d5-84ac-4bd9-a864-27665ffd0f5c
-
-You can install the **Developer Pack** for a specific version of the .NET Framework, if one is available, on all supported platforms.  
-  
-You can install the **Web or Offline installer** on:  
-  
-- Windows 8.1 and earlier  
-  
-- Windows Server 2012 R2 and earlier  
-  
-For a full list, see [System Requirements](~/docs/framework/get-started/system-requirements.md).  
-
-For a general introduction to the .NET Framework for both users and developers, see [Getting Started](../get-started/index.md). For information about deploying the .NET Framework with your app, see the [deployment guide](~/docs/framework/deployment/deployment-guide-for-developers.md). To read about the architecture and key features of the .NET Framework, see the [overview](~/docs/framework/get-started/overview.md).  
-
-<a name="choices"></a>
+|**4.6**|Included in [!INCLUDE[vs_dev14](../../../includes/vs-dev14-md.md)].<br /><br /> [Microsoft .NET Framework 4.6 targeting pack](http://go.microsoft.com/fwlink/?LinkId=528261)|[Download page for 4.6 web installer](http://go.microsoft.com/fwlink/?LinkId=528259)<br /><br /> [Download page for 4.6 offline installer](http://go.microsoft.com/fwlink/?LinkId=528233)|Included in: <br /> Windows 10 <br />[!INCLUDE[vs_dev14](../../../includes/vs-dev14-md.md)]<br /><br /> You can also install on:<br /> Windows 8.1 and earlier<br /> Windows Server 2012 R2 and earlier<br /> (for a full list, see [system requirements](~/docs/framework/get-started/system-requirements.md))|
+|**4.5.2**|[Microsoft .NET Framework 4.5.2 Developer Pack](http://go.microsoft.com/fwlink/?LinkId=397702)<br /><br /> For use with Visual Studio 2013, Visual Studio 2012, or other IDEs|[Download page for 4.5.2 web installer](http://go.microsoft.com/fwlink/p/?LinkId=397703)<br /><br /> [Download page for 4.5.2 offline installer](http://go.microsoft.com/fwlink/p/?LinkId=397706)|You can install on:<br /> Windows 8.1 and earlier<br /> Windows Server 2012 R2 and earlier<br /> (for a full list, see [system requirements](~/docs/framework/get-started/system-requirements.md))|
+|**4.5.1**|[Microsoft .NET Framework 4.5.1 Developer Pack](http://go.microsoft.com/fwlink/?LinkId=324213)<br /><br /> For use with Visual Studio 2013, Visual Studio 2012 or other IDEs|[Download page for 4.5.1 web installer](http://go.microsoft.com/fwlink/p/?LinkId=310158)<br /><br /> [Download page for 4.5.1 offline installer](http://go.microsoft.com/fwlink/p/?LinkId=310159)|Included in:<br /> [!INCLUDE[win81](../../../includes/win81-md.md)]<br /> Windows Server 2012 R2<br /> [Visual Studio 2013](https://my.visualstudio.com/Downloads?q=visual%20studio%202013)<br /><br /> You can also install on:<br /> [!INCLUDE[win8](../../../includes/win8-md.md)] and earlier<br /> [!INCLUDE[winserver8](../../../includes/winserver8-md.md)] and earlier<br />(for a full list, see [system requirements](~/docs/framework/get-started/system-requirements.md))|
```

Comment:
I added Visual Studio 2013 to "for use with" since it says it's included with VS 2013. Not sure if this is correct though so please correct if I'm wrong.